src/main/resources/anniv6Resources/localization/eng/Junkyard/UIstrings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "${ModID}:Junkyard": {
+    "TEXT": [
+      "The Junkyard",
+      "Lots of things to pick up! Gain 2 card rewards per combat but cannot skip card rewards. Also contains unique encounters.",
```

Comment:
I just merged in code that will show the author for each zone. You should add an entry here with the name you'd like to be credited by. You can pull and look at other zones for examples.